ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -376,26 +366,15 @@ export class CohortSearch extends React.Component<Props, State> {
                       select={this.addSelection}
                       selectedIds={selectedIds}
                       selectOption={this.setAutocompleteSelection}
-                      setAttributes={this.setAttributes}
+                      setAttributes={() => {}} // setAttributes no longer needed in V2
```

Comment:
Optional: We can make setAttributes an optional Prop property in tree.component and not pass anything here (not even () => {}, that will make it easier for us to clean later as well :)

Reply:
Good idea, done.